client/components/IndividualChallenge.js

```diff
@@ -16,24 +20,32 @@ const _IndividualChallenge = props => {
   const putValue = isSubmit => {
```

Comment:
Missed this the first time.  Also update this in Run and Submit buttons.
```suggestion
  const updateValue = isSubmit => {
```